content/language/php/deploy.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Test your PHP deployment
+keywords: deploy, php, local, development
+description: Learn how to deploy your application
+---
+
+## Prerequisites
+
+- Complete all the previous sections of this guide, starting with [Containerize
+  a PHP application](containerize.md).
+- [Turn on Kubernetes](/desktop/kubernetes/#turn-on-kubernetes) in Docker
+  Desktop.
+
+## Overview
+
+In this section, you'll learn how to use Docker Desktop to deploy your
+application to a fully-featured Kubernetes environment on your development
+machine. This allows you to test and debug your workloads on Kubernetes locally
+before deploying.
+
+## Create a Kubernetes YAML file
+
+In your `docker-php-sample` directory, create a file named
+`docker-php-kubernetes.yaml`. Open the file in an IDE or text editor and add
+the following contents. Replace `DOCKER_USERNAME/REPO_NAME` with your Docker
+username and the name of the repository that you created in [Configure CI/CD for
+your PHP application](configure-ci-cd.md).
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: docker-php-demo
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      hello-php: web
+  template:
+    metadata:
+      labels:
+        hello-php: web
+    spec:
+      containers:
+      - name: hello-site
+        image: DOCKER_USERNAME/REPO_NAME
+        imagePullPolicy: Always
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: php-entrypoint
+  namespace: default
+spec:
+  type: NodePort
+  selector:
+    hello-php: web
+  ports:
+  - port: 80
+    targetPort: 80
+    nodePort: 30001
+```
+
+In this Kubernetes YAML file, there are two objects, separated by the `---`:
+
+ - A Deployment, describing a scalable group of identical pods. In this case,
+   you'll get just one replica, or copy of your pod. That pod, which is
+   described under `template`, has just one container in it. The container is
+   created from the image built by GitHub Actions in [Configure CI/CD for your
+   PHP application](configure-ci-cd.md).
+ - A NodePort service, which will route traffic from port 30001 on your host to
+   port 80 inside the pods it routes to, allowing you to reach your app
+   from the network.
+
+To learn more about Kubernetes objects, see the [Kubernetes documentation](https://kubernetes.io/docs/home/).
+
+## Deploy and check your application
+
+1. In a terminal, navigate to the `docker-php-sample` directory
+   and deploy your application to Kubernetes.
+
+   ```console
+   $ kubectl apply -f docker-php-kubernetes.yaml
+   ```
+
+   You should see output that looks like the following, indicating your Kubernetes objects were created successfully.
+
+   ```shell
```

Comment:
I think `shell` == `bash` in hugo's syntax highlighting (chroma). I think we should use `text` for these instead
https://gohugo.io/content-management/syntax-highlighting/#list-of-chroma-highlighting-languages